sync_issues_to_jira/sync_to_jira.py

```diff
@@ -53,7 +51,15 @@ def main():
         event = json.load(f)
         print(json.dumps(event, indent=4))
 
-    event_name = os.environ['GITHUB_EVENT_NAME']  # The name of the webhook event that triggered the workflow.
+    event_name = os.environ['GITHUB_EVENT_NAME']
+
+    # Check if event is workflow_dispatch and action is mirror issues. If so, run manual mirroring and skip rest of the script. Works both for issues and pull requests.
+    input_action = event['inputs']['action']
```

Comment:
Won't this crash when `action` is not defined?

Reply:
Fixed in last commit [3e91c76](https://github.com/espressif/github-actions/pull/37/commits/3e91c76a50960b4dc04a3b42f64bfe3cf1f3591b)